libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -1487,6 +1487,12 @@ class NavEKF3_core : public NavEKF_core_common
     bool EKFGSF_run_filterbank;             // true when the filter bank is active
     uint8_t EKFGSF_yaw_valid_count;         // number of updates since the last invalid yaw estimate
 
+    // logging timestamps
```

Comment:
How about moving these members down to the bottom of the .h file just below the logging methods?

As a side note, it looks a bit like Paul always put the methods together at the top and then members all together at the bottom but that pattern has been broken by other devs.  I don't want to be too annoying about it but with a complex system it's nice to follow a pattern.

Reply:
FWIW I'd like us to group member variables with related member methods where possible.

Reply:
@peterbarker,

Sure, I'm happy with that too.  As per usual, I don't mind too much what the specific pattern is, I just like us to have a pattern.  Tidy code is easier for devs to get their head around.  If you have to move back and forth around in the file, it's a waste of time.

Reply:
this is scope creep. The place I've put these variables fits the existing pattern. If you want a different pattern then propose a PR to rearrange the code and ask Paul to review

Reply:
I'm just suggesting moving them down 20 lines so they're with the logging methods which will make it a bit prettier and logical.

The "existing pattern" is just what you've done for the last couple of EKF changes you've made which also didn't follow the existing pattern.

The change is not a big deal, the push back against such a small change request is disappointing.